### ISSUE TITLE
URL to use for the batch transcription sample with custom language model

### ISSUE THREAD

**Issue by Damally** [Azure-Samples/cognitive-services-speech-sdk#197]
Hi guys,

I am trying to use the sample batch transcription csharp sample to transcribe the .wav file in a blob container. Am getting the error invalid Host and the sample stop at the breakpoint.

Question is .. when I have a custom language model specified in Australiaeast.cris.ai (the swagger page returned a status 202 when I test it), which URL should I be using ? Should I be using The cris (which there’s a /transcription as well) 

Or

https://australiaeast.stt.speech.microsoft.com/speech/recognition/conversation/cognitiveservices/v1

According to doc all the speech services use the same api since mar2019 but the doc refers to the cris.ai for custom service, yet the doc also mentioned migration of custom service to speech api. It’s really confusing ..

So in short, how do one reference the speech to text api when a custom language model is involved, when using the Rest api for batch transcription?

Thank you for any light that you can shed into this 😊

**Comment by PanosPeriorellis** [comment]
hi Damally, 

You need a paying key to use Batch Transcription. So in the Azure Portal, create a Speech resource with an S0 key for region australiaeast, create your custom models with that key on https://australiaeast.cris.ai portal or using the Swgger at https://australiaeast.cris.ai/swagger/ui/index 

Make sure you pass the right key in your transcription request along with the SAS Uri of your file and the custom model that you wish to use.

regards
Panos

**Comment by Damally** [comment]
Thanks @Panos,

I’ve checked to make sure that I am using the S0 key (the pay tier) as i researched in forum as well as checking the documentation. That’s the ‘speech key’ that I am referring to. not bing-speech, not the cognitive-service (the new all in one except speech) either

**Comment by wolfma61** [comment]
closing
